File: src/main/java/com/ggemo/bilidanmakuclient/http/response/impl/DanmakuServerConfResponse.java
package com.ggemo.bilidanmakuclient.http.response.impl;

import com.alibaba.fastjson.JSON;
import com.alibaba.fastjson.annotation.JSONField;
import com.ggemo.bilidanmakuclient.http.response.AbstractResponse;
import com.ggemo.bilidanmakuclient.http.response.responsedataa.DanmakuServerConfResponseData;
import com.ggemo.bilidanmakuclient.http.response.responsedataa.RoomInitResponseData;

import java.util.List;
import java.util.Map;

public class DanmakuServerConfResponse extends AbstractResponse<DanmakuServerConfResponseData> {
    public DanmakuServerConfResponse(int code, String message, DanmakuServerConfResponseData data) {
        super(code, message, data);
    }

    public static DanmakuServerConfResponse parse(String json) {
        DanmakuServerConfResponse res = JSON.parseObject(json, DanmakuServerConfResponse.class);
        return res;
    }
}

File: blive-danmaku-client-jvm/src/main/kotlin/com/ggemo/va/blivedanmakuclient/httpclient/httpresponse/impl/DanmakuServerConfResponse.kt
package com.ggemo.va.bilidanmakuclient.http.response.impl

import com.ggemo.va.bilidanmakuclient.http.response.AbstractResponse
import com.alibaba.fastjson.JSON
import com.ggemo.va.bilidanmakuclient.http.response.responsedataa.DanmakuServerData

class DanmakuServerConfResponse :
    AbstractResponse<DanmakuServerData?>() {
    companion object {
        fun parse(json: String?): DanmakuServerConfResponse {
            return JSON.parseObject(json, DanmakuServerConfResponse::class.java)
        }
    }
}